### PR TITLE
Joi Sanitising

### PR DESCRIPTION
**Pull request by Jordan141**
Seeing if Joi is a superior option to the completed but unmerged PR in #250

### CODE REVIEW

**Comment by michaelKurowski** [comment]
How is it going so far @Jordan141

**Comment by Jordan141** [comment]
Ready for review but needs testing

File: app/public/scripts/utils.js [hunk 1]
```diff
@@ -206,9 +208,20 @@ function analyticsFingerprintSender() {
...
   const targetPage = parseInt(loadMoreLink.dataset.page)
   const route = location.pathname
   const params = new URLSearchParams(location.search)
   params.set('page', targetPage)
   loadMoreLink.href = route + '?' + params.toString()
+}
+
+function onSubmitListener() {
+  const allInputs = document.getElementsByTagName('input')
+  for(let i = 0; i < allInputs.length; i++) {
```
Comment (michaelKurowski) [inline app/public/scripts/utils.js:221]: `allInputs.forEach`

**Review by michaelKurowski — APPROVED** [review]
one finding, approved in advance

**Pull request merged**